src/index.js

```diff
@@ -47,6 +48,7 @@ export default {
         'jsdoc/check-alignment': 'warn',
         'jsdoc/check-examples': 'off',
         'jsdoc/check-indentation': 'off',
+        'jsdoc/check-lines-alignment': 'warn',
```

Comment:
This should be "off" for now (noting that when enabled, it will be for the "never" option).

Reply:
Oops! Missed that. Updated here: https://github.com/gajus/eslint-plugin-jsdoc/pull/636/commits/1c1dd685b5a16a8f3b5660b68447502500040c9a